### PR TITLE
assign agent to ticket

### PR DESCRIPTION
**Pull request by tejaswinichile**
Issue #328

### CODE REVIEW

File: app/javascript/Admin/Admin/Data/Ticket.elm [hunk 1]
```diff
@@ -88,6 +114,13 @@ commentObject =
         |> GQLBuilder.with (GQLBuilder.field "info" [] GQLBuilder.string)
 
 
+agentObject : GQLBuilder.ValueSpec GQLBuilder.NonNull GQLBuilder.ObjectType Agent vars
```
Comment (tejaswinichile) [inline app/javascript/Admin/Admin/Data/Ticket.elm]: @breezyboa I want to make this object `Nullable`. I had declared this as `Nullable` but it's giving issue further. 
So what I want is, `agentObject` for the ticket might be not present. How should I handle this?

**Comment by breezyboa** [comment]
@tejaswinichile _a please resolve conflicts

**Comment by tejaswinichile** [comment]
@breezyboa _a Added last 2 commits according to what we have discussed to set Nullable value for ticket's agent.

**Comment by breezyboa** [comment]
@tejaswinichile _a please resolve conflicts and merge

**Comment by tejaswinichile** [comment]
@breezyboa _a Please check.

**Comment by tejaswinichile** [comment]
@breezyboa Resolved conflicts, merging this PR.

**Pull request merged**